test/block-test.js

```diff
@@ -4,19 +4,43 @@
 
 'use strict';
 
-const {BloomFilter} = require('bfilter');
 const assert = require('bsert');
-const common = require('./util/common');
 const Block = require('../lib/primitives/block');
-const MerkleBlock = require('../lib/primitives/merkleblock');
-const consensus = require('../lib/protocol/consensus');
-const Network = require('../lib/protocol/network');
-const Script = require('../lib/script/script');
-const bip152 = require('../lib/net/bip152');
-const CompactBlock = bip152.CompactBlock;
-const TXRequest = bip152.TXRequest;
-const TXResponse = bip152.TXResponse;
+const MemBlock = require('../lib/primitives/memblock');
 
 describe('Block', function() {
   this.timeout(10000);
+
+  describe('Serialization', function() {
+    let block = null;
+    let raw = null;
+
+    it('should create a block from options', async () => {
+      block = new Block({
+        version: 1,
+        prevBlock: Buffer.alloc(32, 0),
+        merkleRoot: Buffer.alloc(32, 1),
+        witnessRoot: Buffer.alloc(32, 2),
+        treeRoot: Buffer.alloc(32, 3),
+        reservedRoot: Buffer.alloc(32, 4),
+        time: 0,
+        bits: 0,
+        nonce: 0,
+        extraNonce: Buffer.alloc(24, 5),
+        mask: Buffer.alloc(32, 6)
+      });
+      assert(block);
+    });
+
+    it('should deserialize and reserialze block', async () => {
+      raw = block.toRaw();
```

Comment:
Nit: use `encode`/`decode` over `toRaw` and `fromRaw`

Reply:
thanks updated